lisa/transformers/hyperv_preparation.py

```diff
@@ -0,0 +1,64 @@
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional, Type
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema
+from lisa.node import quick_connect
+from lisa.tools import PowerShell
+from lisa.transformer import Transformer
+from lisa.util import field_metadata
+
+
+@dataclass_json
+@dataclass
+class HyperVPreparationTransformerSchema(schema.Transformer):
```

Comment:
Refer to `DisableCloudComponentsTransformerSchema` to reuse `DeploymentTransformerSchema`.

Reply:
Modified as suggested.